rpc/wrtc_signaling_server.go

```diff
@@ -171,70 +200,27 @@ func (srv *WebRTCSignalingServer) Call(req *webrtcpb.CallRequest, server webrtcp
 	}
 	defer sendCancel()
 
-	defer func() {
-		var errToSend error
-
-		if callErr == nil {
```

Comment:
Notable behavior change is that this `callErr == nil` block will never result in calling `SendOfferError`.